docs/de/erste-schritte/performance-messen.md

```diff
@@ -1,3 +1,57 @@
 ---
 title: Performance Dashboard
+todo: Bilder müßen noch ausgetauscht werden
 ---
+
+Um die Wertentwicklung Deines Portfolios zu beurteilen, solltest Du Dir zunächst das Dashboard samt den Dir zur Verfügung stehenden Dashboards ansehen. Die wichtigsten Wertentwicklungs- und Risikoindikatoren für das gesamte Portfolio werden in einem Dashboard zusammengefasst, das über das Menü `Ansicht --> Berichte --> Performance` oder über die Seitenleiste geöffnet werden kann. (siehe Abbildung 1). Eine umfassende Übersicht über das Dashboard findest Du im [Referenzhandbuch](../referenzhandbuch/ansichten/berichte/performance/index.md) unter `Ansicht --> Berichte --> Performance`. Wie immer zeigen grüne Werte einen Gewinn an, während rote Werte einen Verlust bedeuten.
+
+Abbildung: Dashboard mit unterschiedlichen Kennzahlen für einen zweijährigen Berichtszeitraum.{class=pp-figure}
+
+![](../how-to/historische-aktienkurse/images/performance-dashboard-2yrs.png)
```

Comment:
Hallo @Sn1kk3r5 
super Arbeit bisher.... Rechtschreibung... Grammatik.... sehr geil! Saubere Arbeit !! 👍🏻 

Eine möchte nochmal den Hinweis geben, dass die Bilder (Screenshots) einheitlich sein sollten.
In Portfolio Performance bitte das "Helle Layout" verwenden, da der "Dark Mode" in den unterschiedlichen Betriebssystemen sich sehr unterscheidet. Die meisten Übereinstimmungen sind eben im "Hellen Modus", was die Standardeinstellung ist. 
Auch bitte darauf achten, dass Ihr deutsche und englische Screenshots nicht vertauscht.

Gruß
Alex #

Reply:
Hi @Nirus2000,

danke fürs Feedback. In der finalen Version wird nur noch das helle Layout verwendet werden. Hugo war das seiner Zeit nicht bewusst. Für ein einheitliches Layout pflege ich parallel das Kommer.xml in Absprache mit Andreas. Das macht die Screenshots samt Übersetzung erheblich leichter. Bilder, die du heute im deutschen Manual auf Englisch siehst, sind zurzeit noch mit Absicht, um überhaupt einen Eindruck zu haben. Werden aber natürlich komplett ersetzt.

Viele Grüße